tests/framework/installer/ceph_installer.go

```diff
@@ -101,6 +104,13 @@ func (h *CephInstaller) CreateCephOperator(namespace string) (err error) {
 		}
 	}
 
+	if err = h.WipeClusterDisks(namespace); err != nil {
```

Comment:
Can you create a helper method that retries? This same code block is called in two places.